aot/src/main/java/com/dylibso/chicory/experimental/aot/AotMachine.java

```diff
@@ -1,19 +1,24 @@
 package com.dylibso.chicory.experimental.aot;
 
+import com.dylibso.chicory.compiler.CompilerMachine;
+import com.dylibso.chicory.compiler.internal.MachineFactory;
 import com.dylibso.chicory.runtime.Instance;
 import com.dylibso.chicory.runtime.Machine;
 import com.dylibso.chicory.wasm.ChicoryException;
 
 /**
- * Machine implementation that AOT compiles function bodies.
+ * Machine implementation that compiles function bodies.
  * All compilation is done in a single compile phase during instantiation.
+ *
+ * @deprecated This class is deprecated, use {@link CompilerMachine} instead.
  */
+@Deprecated
```

Comment:
I'd avoid the deprecation here, we are coming from an experimental module and we have not been guaranteeing  compatibility on purpose.
I'd invest in having good release notes with clear migration guidelines.

Reply:
Maintaining this little file is not much effort.   So I would vote to keep this around until 2.0 is out.   Then users would have had ample time and the Warnings to fix the deprecation.

Reply:
@chirino would you mind confirming this is the only file containing deprecations?
The name of the dependency is going to change, but, maintenance of this one is really low.

Reply:
Yep. that's it.

Reply:
Looks good :+1: